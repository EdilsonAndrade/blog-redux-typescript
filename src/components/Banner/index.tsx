import React from 'react';
import { Container } from './styles';

const Banner: React.FC = () => (
  <Container>
    <h1>PodVisory Blog</h1>
  </Container>
);

export default Banner;
