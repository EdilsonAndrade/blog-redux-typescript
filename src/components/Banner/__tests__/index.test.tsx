import renderer from 'react-test-renderer';
import Banner from '../';

describe("[Banner Component]", () => {
    it('renders banner correctly', () => {
        const tree = renderer
            .create(<Banner />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });

})
