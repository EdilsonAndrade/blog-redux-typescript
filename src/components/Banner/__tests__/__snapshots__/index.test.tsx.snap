// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Banner Component] renders banner correctly 1`] = `
<div
  className="sc-bdfBQB dnmJyo"
>
  <h1>
    PodVisory Blog
  </h1>
</div>
`;

exports[`renders banner correctly 1`] = `
<div
  className="sc-bdfBQB dnmJyo"
>
  <h1>
    PodVisory Blog
  </h1>
</div>
`;

exports[`renders correctly 1`] = `
<div
  className="sc-bdfBQB dnmJyo"
>
  <h1>
    PodVisory Blog
  </h1>
</div>
`;
